docs/sources/flow/reference/stdlib/trim.md

```diff
@@ -0,0 +1,28 @@
+---
+aliases:
+- ../../configuration-language/standard-library/trim/
+title: trim
+---
+
+# trim
+
+`trim` removes the specified set of characters from the start and end of the given string.
+
+```
+trim(string, str_character_set)
+```
+
+Every occurrence of a character in the second argument is removed from the start and end of the string specified in the first argument.
```

Comment:
Is this a restatement of the definition? I'm not sure why this is here... none of the other definitions have this.

Reply:
Seems like an left over. I remove this.